src/recaptool

```diff
@@ -41,127 +44,172 @@ if [[ -r "${CONFFILE}" ]]; then
 fi
 
 memproc() {
-	PROCESS="${1}"
-	TMP=$( mktemp )
-	>"${TMP}"
-	for i in $( ls "${BASEDIR}"/ps_*.log ); do
-		head -n 1 "${i}" | tr "\n" "\t" >> "${TMP}"
-		egrep "${PROCESS}" "${i}" -c | tr "\n" "\t" >> "${TMP}"
-		egrep "${PROCESS}" "${i}" | awk '{ SUM += $6 } END {print SUM/1024, "M"};' >> "${TMP}"
-	done
-	sort "${TMP}"
-	rm -f "${TMP}"
+  set -x
```

Comment:
Any reason why `set -x` is enabled here and disabled [on line 58](https://github.com/rackerlabs/recap/pull/180/commits/83fe86223356d47e7e41d69db32b7ef1772c82c3#diff-51a334d558375221908afaef23c51bb5R58)? was this for debug purposes and made it into the commit?